@media/NFTCard2/NFTCard2.tsx

```diff
@@ -0,0 +1,167 @@
+import { ImageElement } from 'components'
+import { Link } from 'components/Link'
+import {
+  ImageEncodingTypes,
+  NounsTokensByOwnerAddressQuery,
+} from 'types/zora.api.generated'
+
+import { useMemo } from 'react'
+
+import { NFTCardMarket } from '@market'
+import { CollectionThumbnail } from '@media/CollectionThumbnail'
+import {
+  cardImageWrapper,
+  cardWrapper,
+  titleHeading,
+  titleScroll,
+  titleWrapper,
+} from '@media/NftMedia.css'
+import { useRawImageTransform } from '@media/hooks/useRawImageTransform'
+import { FallbackThumbnail } from '@noun-auction'
+import { useIsOwner } from '@shared'
+import { Box, Flex, Heading, Separator, Stack } from '@zoralabs/zord'
+
+/** 
+  tokenId
+  collectionAddress
+  collectionName
+  tokenOwner 
+  token?.image?.url or token?.image?.mediaEncoding?.poster
+*/
+
+type ImageType = NounsTokensByOwnerAddressQuery['tokens']['nodes'][0]['token']['image']
+type Props = {
+  collectionAddress: string
+  tokenId: string
+  collectionName?: string
+  tokenName?: string
+  // optionally set this as flag for /manage page (where we know for sure that user is owner)
+  isOwner?: boolean
+  image: ImageType
+}
+
+export function NFTCard2({
```

Comment:
Great that this eliminates the useNFTProvider() calls, which seemed to me like a super weird pattern.